explainer.md

```diff
@@ -346,13 +522,88 @@ of custom-drawn APIs.
 On the web, this would allow creating an accessible solution to canvas-based UI
 which does not rely on fallback or visually-hidden DOM content.
 
-The API for Virtual Accessibility Nodes will be specified at a later date.
+Constructing a node can be achieved simply by creating an AccessibleNode
```

Comment:
Right, I mean that putting official-looking details in here might invite bikeshedding, not that adding them in constitutes bikeshedding.

Can we word this in a way that indicates the direction without providing any bikeshed-worthy code examples?